chapter4/src/main/java/com/backstopmedia/kotlin/chapter4/api/KTwitterApiClient.kt

```diff
@@ -0,0 +1,30 @@
+package com.backstopmedia.kotlin.chapter4.api
+
+import com.twitter.sdk.android.core.TwitterApiClient
+import com.twitter.sdk.android.core.TwitterSession
+import com.twitter.sdk.android.core.models.Tweet
+import com.twitter.sdk.android.core.models.User
+import retrofit.http.GET
+import retrofit.http.Query
+import rx.Observable
+
+/**
+ * Created by Tudor Luca on 14/10/15.
+ */
+class KTwitterApiClient(val session: TwitterSession) : TwitterApiClient(session) {
+
+    val kTwitterApi: KTwitterApi by lazy { getService(KTwitterApi::class.java) }
+}
+
+interface KTwitterApi {
```

Comment:
Looks like the home timeline is built into the SDK, but I guess if we want an observable this is probably the easiest way to do it.

Really wish Retrofit 2.0 was out so we could just add a bunch of extensions to the `Call` object :-(